### ISSUE TITLE
[Bug] pause switch does not resume print

### ISSUE THREAD

**Issue by manolitto** [cmroche/OctoPrint-HomeAssistant#96]
### Expected behaviour
Switching pause switch to off should actually resume the print in OctoPrint.

### Observed behaviour
Switching pause switch to off has no effect.

### What I found out
When the printer is in pause mode and when switching the pause switch in HA to off a MQTT message with payload "False" is sent to topic "octoPrint/hassControl/pause" which seems ok.
However, it does NOT actually resume the print on OctoPrint side.

What DID work is sending an empty message to "octoPrint/hassControl/pause" - it successfully resumed my print.

**Comment by cmroche** [comment]
Thanks for the report, I'll take a look soon.

**Comment by nquick** [comment]
Hi,

Did you find a way of resolving this? I'm building a WiFi filament sensor and can get it to pause, but not resume. 

Thanks

**Comment by AlekEagle** [comment]
Hey, I made a PR fixing it, depending on the switch position, Lovelace will send `True` or `False` for the MQTT message/payload. If the switch is currently set to off and the user interacts with it, Lovelace will emit `True` via MQTT, and vice versa for the switch in the on position. Also, for button entities, Lovelace sends `PRESS` as the payload. What I did in #100 was check if the message is `PRESS` for the buttons. I have tested this and it does work with the latest Home Assistant and OctoPrint.
